bin/document-signers.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+
+/*
+ * to run document-signers applet from the project root folder type in your console:
+ * > node bin/document-signers <client_id> <client_secret> <username> <password> <document_id>
+ * <client_id>, <client_secret>, <username>, <password>, <document_id> - are required params
+ * options main flags:
+ *  --freeform-invites
+ *  --field-invite-status
+ *  --payment-request-status
+ *
+ * additional options flags for --field-invite-status
+ *  all,pending,declined,fulfilled,created,skipped
+ *  exp.(--field-invite-status=pending,declined)
+ *
+ * additional options flags for --payment-request-status
+ *  all,created,pending,skipped,fulfilled
+ * exp.(--payment-request-status=declined,all)
+ */
+
+'use strict';
+const args = process.argv.slice(2);
+const flags = args.filter(arg => /^--/.test(arg));
+const params = args.filter(arg => !/^--/.test(arg));
+
+const freeForm = 'freeFormInvites';
+const fieldStatus = 'fieldInviteStatus';
+const payment = 'paymentRequestStatus';
+const allExceptFirst = arr => arr.slice(1).join('').split(',');
+
+const options = flags
```

Comment:
I have an idea how to refactor it. I'll will suggest when you will be applying changes